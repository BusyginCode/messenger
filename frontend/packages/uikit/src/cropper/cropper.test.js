import React from 'react'
import { shallow, mount } from 'enzyme'
import Cropper from './cropper'

describe('uikit/Cropper', () => {
  it('should render correctly', () => {
    expect(mount(<Cropper src="http://image.jpg" />)).toMatchSnapshot()
  })
  it('should render empty correctly', () => {
    expect(shallow(<Cropper />)).toMatchSnapshot()
  })
})
