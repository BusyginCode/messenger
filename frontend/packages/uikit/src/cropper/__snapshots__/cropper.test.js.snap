// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Cropper should render correctly 1`] = `
<Cropper
  src="http://image.jpg"
>
  <ReactCropper
    aspectRatio={1}
    autoCropArea={0.1}
    background={true}
    data={null}
    dragMode="move"
    enable={true}
    minCanvasHeight={150}
    minCanvasWidth={150}
    minContainerHeight={150}
    minContainerWidth={150}
    minCropBoxHeight={150}
    minCropBoxWidth={150}
    modal={true}
    movable={false}
    rotateTo={0}
    scaleX={1}
    scaleY={1}
    src="http://image.jpg"
    style={
      Object {
        "maxHeight": 300,
        "maxWidth": 500,
      }
    }
    viewMode={1}
    zoomTo={1}
    zoomable={false}
  >
    <div
      alt={null}
      crossOrigin={null}
      src={null}
      style={
        Object {
          "maxHeight": 300,
          "maxWidth": 500,
        }
      }
    >
      <img
        alt="picture"
        src="http://image.jpg"
        style={
          Object {
            "opacity": 0,
          }
        }
      />
    </div>
  </ReactCropper>
</Cropper>
`;

exports[`uikit/Cropper should render empty correctly 1`] = `""`;
